src/tools/interop/idt/Dockerfile

```diff
@@ -0,0 +1,58 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+FROM debian:bookworm
```

Comment:
I don't understand thge connection between this Dockerfile and Kali... Is this Dockerfile trying to build a container that contains the tools that are pre-installed on Kali? Or these are just the host tools needed to process the monitor information?

Reply:
The sole reason for suggesting the Kali Base image was to ensure nexmon patches are installed for placing the inbuilt WiFi card in monitor mode. We then pass interfaces from the host to the container. Otherwise, I'd rather not suggest it.

The container is meant to contain everything required for execution and processing of capture / discovery.

Reply:
It will be nice to remove Kali from the equation; we'll get to this!